### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Achleshwar**

### CODE REVIEW

**Comment by Achleshwar** [comment]
added sam installation command

File: README.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ Dynamatte: A dynamic matting method to generate in scene video fusion
 +Achleschwar
 $Chris
 
+## SAM Installation
+```
+pip install git+https://github.com/facebookresearch/segment-anything.git
```
Comment (klammecr) [inline README.md:24]: Good to add. Added this to the requirements.txt file as well so may be redundant

**Review by klammecr — APPROVED** [review]
See comment, fine with adding

File: README.md (hunk 1 above)
Comment (Achleshwar) [reply README.md:24]: yup agreed

**Pull request closed without merging**